modules/nw-udn-cr.adoc

```diff
@@ -0,0 +1,97 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-udn-cr_{context}"]
+= Creating a UserDefinedNetwork custom resource
+
+The following procedure sets up a user-defined network that is namespace scoped.
+
+.Prerequisites
+* You must
+.Procedure
+
+[NOTE]
+====
+If any cluster default networked pods exist before the user-defined network is created, any further pods created in this namespace will return an error message: `What_is_this`.
+====
+
+. Create a request for a user-defined network:
+
+.. Create a YAML file, such as `my-udn-request.yaml`, to define your request with content like the following example:
```

Comment:
```suggestion
.. Create a YAML file, such as `my-udn-request.yaml`, to define your request with content as in the following example:
```

Reply:
Style guide complains about the "like", so I tried "as in". You've already used "such as" once and it would look odd to use it again.